belvedere/src/main/java/zendesk/belvedere/BuildVersionProvider.java

```diff
@@ -0,0 +1,14 @@
+package zendesk.belvedere;
+
+/**
+ * Provides the build version of the current Android device
+ */
+interface BuildVersionProvider {
+
+    /**
+     * Returns the build version of the current android device. Usually this will be {@link android.os.Build.VERSION#SDK_INT}.
+     *
+     * @return the build version of the current Android device
+     */
+    int currentVersion();
```

Comment:
Method names should be verbs. Maybe rename to `getCurrentVersion`?

Reply:
I have the feeling this interface is a bit of an overkill given that it's only used in one place. Could you achive the same behavior by passing in the version number into the constructor of `ImageStreamCursorProvider`?

Reply:
Can certainly try!

Reply:
Its a fair point. At the least we don't actually need the Interface I think, the single implementation is fine as we can still mock it for tests.

Reply:
Hello btw!

Reply:
👋

Reply:
Removed the interface and updated tests